modules/nf-core/phispy/tests/main.nf.test

```diff
@@ -0,0 +1,67 @@
+// nf-core modules test phispy
+nextflow_process {
+
+    name "Test Process PHISPY"
+    script "../main.nf"
+    process "PHISPY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "phispy"
+
+    test("bacteroides_fragilis - gbff") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['bacteroides_fragilis']['genome']['genome_gbff_gz'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            def log_text = path(process.out.log[0][1]).text
```

Comment:
I just want to point out there is also the `with` method to do this.
https://nf-co.re/docs/contributing/tutorials/nf-test_assertions#using-with

Reply:
Thanks for the suggestion! I had tried it when I first implemented the tests, but it hadn't worked for some reason (guess I was doing something wrong). Done in [badb93c](https://github.com/nf-core/modules/pull/5292/commits/badb93c90622c42f30ad4661562bfa236aadf3ac).